pyro/distributions/transforms/__init__.py

```diff
@@ -32,7 +32,7 @@
 from .block_autoregressive import BlockAutoregressive, block_autoregressive
 from .cholesky import (
     CholeskyTransform,
-    CorrLCholeskyTransform,
```

Comment:
I'd lean towards preserving the old import so as to avoid breaking existing code, but I'll defer to your judgement if you think we should delete both.  (I think of "deprecation" as meaning "it still works but we no longer recommend it", whereas removing from `__init__.py` would be a truly breaking change)